### ISSUE TITLE
More detailed time of day for warnings

### ISSUE THREAD

**Issue by marty2022** [corona-warn-app/cwa-wishlist#746]
## Current Implementation
If you have been near a positive tested person for a longer time you get a warning message with day, month and year when the encounter happened.

## Suggested Enhancement
Beside the date also the time of the day like morning, forenoon, midday, afternoon, evening, night should be displayed so one can better estimate the risk.

## Expected Benefits
As I got the warning I was unable to estimate my own risk:
- Was it in the morning when using public transports while wearing a mask?
- Was it in the forenoon during work while not wearing a mask from the same room, meetings, neighbour rooms or even other companies on the same floor?
- Was it at midday when eating in the cantina while not wearing a mask?
- Was it in the afternoon during work while not wearing a mask from the same room, meetings, neighbour rooms or even other companies on the same floor?
- Was it in the evening when using public transports or shopping while wearing a mask?
- Was it at the night from my neighbours?

So when I would have known a more specific timeframe I would be able to estimate my personal risk much more better and I am also able to specific warn other contact persons.

---
Duplicate of #266
Internal Tracking ID: [EXPOSUREAPP-3670](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3670)

**Comment by cvoinf** [comment]
...
> 
> > If I only meet one person on a day where I get a positive warning I can also identify this person with the current implementation.
> 
> Sure, in real life scenario, you probably now this person and had been informed by him directly anyway.
Same true for a person that I can identify in an hour timeslot.
> 
> > It's always a balance between privacy and benefit.
> 
> That is exactly the point, and current view is that the offered solution is the "right" balance. Of course to find this balance and to agree on common terms is not an easy task. 

Correct, this task is not easy, but the "right" balance might change with the time. At the beginning of this pandemic being identified as corona positive probably was more stigmatic as now with incidence levels of 400+. So while the decision perhaps was correct a year ago, one might come to other conclusions now.
 


> Note that the current ENF by Apple and Google (which limits the functionality to the day of the encounter, and time is unknown by the tracing apps which make use of the OS ENF) is a transnational solution, and many considerations had to go into the definition to achieve acceptance und a working system.

Perhaps this would be a good point to overthink the dependence on google anyway. CCTG has proven the same transnational solution works without proprietary code, so why is CWA still not 100% open source?

**Comment by cvoinf** [comment]
> @cvoinf Awesome, thank you very much for these links. Too bad that Warn-App-Companion needs root-permissions (or the Corona-Contact-Tracing-App which is not so well tested as they say).

No, my phone is not rooted. You can export the database from cctg to the companion-app directly.

**Comment by cvoinf** [comment]
Another suggestion: 

The red bar marking I had contact to somebody who shared his status of infection stays red for 14 days, then flips to green the day after. 

This does not reflect the risk of being infected. The longer the meeting is in the past, the less probable is my own risk of infection. Same true for the duration of the meeting and the attenuation of the signal.

Thus the state of risk should not only be black/white (or red/green) but a value on a scale, that diminishes with time since the meeting is more in the past. 

![scale](https://user-images.githubusercontent.com/11172467/148737403-3e3497ce-55b6-43bb-86fa-bbd861aafa7e.jpg)
